test/FunctionalTests/Client/UnaryTests.cs

```diff
@@ -68,6 +68,35 @@ Task<HelloReply> UnaryThrowError(HelloRequest request, ServerCallContext context
         StringAssert.StartsWith("Failed to deserialize response message.", call.GetStatus().Detail);
     }
 
+    [TestCase("fr", "fr")]
```

Comment:
IMO we should have a test of what happens if the culture requested is not supported by the server

Reply:
That's testing the localization feature. These tests are to double-check that gRPC doesn't lose the culture that localization sets.